hack/local-up-kubeedge.sh

```diff
@@ -36,7 +36,13 @@ function check_prerequisites {
   check_kind
   if [[ "${CONTAINER_RUNTIME}" = "docker" ]]; then
     # if we will use docker as edgecore container runtime, we need to verify whether docker already installed
-    verify_docker_installed
+    verify_cri_dockerd_installed
```

Comment:
```suggestion
    verify_docker_installed 
    verify_cri_dockerd_installed
```

Both docker and cri-dockerd are needed to verify.  Or let's remove these modifications related to Docker for now. As mentioned earlier, let's exclude Docker-related changes in this pull request.